src/components/AddItemForm.jsx

```diff
@@ -18,8 +18,8 @@ export default function AddItemForm({ listPath, data }) {
 	const [itemDuration, setItemDuration] = useState(7);
 
 	const normalizedItemNames = useMemo(() => {
-		return data.map((item) => normalizeInput(item.name));
-	}, [data]);
+		return data.data.map((item) => normalizeInput(item.name));
+	}, [data.data]);
```

Comment:
What changed so that now we have data inside of data?

Reply:
Hey @aberger3647 , if you look in the changes to firebase.js for the `useShoppingListData` function we are now returning two fields, data and loading `return { data, loading };`
This is to account for the [loading state issue](https://github.com/orgs/the-collab-lab/projects/200?pane=issue&itemId=55172963) in the list view.
The useShoppingListData was running before listPath was being passed in, which caused the issue you see in the bug ticket. 
Here I am just setting a loading field to either true when listPath is passed in, or false when no listPath exists so that the list view only loads when there's actually a list to display the data for. I think maybe it makes more sense to eventually update it to `return {items, loading}` since `data.data` may be a bit confusing.

Reply:
Makes sense! Ty for the explanation.